web-bundle/src/main/java/com/graphhopper/replica/StreetEdgeExporter.java

```diff
@@ -245,16 +237,26 @@ public static void writeStreetEdgesCsv(GraphHopper configuredGraphHopper,
         }
     }
 
+    private static String parseWayTag(long osmId, Map<Long, Map<String, String>> osmIdToWayTags, String wayTag) {
```

Comment:
nit: i see we wrap calls to this method in `Optional.ofNullable`, so we could just have it return an `Optional<String>` instead?

Reply:
Sure, that's cleaner! Updated